### PR TITLE
Fix docGen and update configuration/metrics

### PR DESCRIPTION
**Pull request by jja725**
### What changes are proposed in this pull request?
Fix docGen and update configuration/metrics

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

File: docs/_data/table/common-configuration.csv [hunk 1]
```diff
@@ -210,12 +209,6 @@ alluxio.zookeeper.leader.connection.error.policy,"SESSION"
 alluxio.zookeeper.leader.inquiry.retry,"10"
 alluxio.zookeeper.leader.path,"/alluxio/leader"
 alluxio.zookeeper.session.timeout,"60s"
-aws.accessKeyId,""
-aws.secretKey,""
```
Comment (ZhuTopher) [inline docs/_data/table/common-configuration.csv:209]: The following is still in `common-configuration.yaml`
```
aws.accessKeyId:
  'The access key of S3 bucket.'
aws.secretKey:
  'The secret key of S3 bucket.'
```

Should they be removed?

**Review by ZhuTopher — CHANGES_REQUESTED** [review]
Overall LGTM, not sure about if removing `aws.accessKeyId` and `aws.secretKey` is correct or not though.

File: docs/_data/table/common-configuration.csv (hunk 1 above)
Comment (Xenorith) [reply docs/_data/table/common-configuration.csv:209]: check against one of the previous release branches? ex. release-2.6.2

Comment (jja725) [reply docs/_data/table/common-configuration.csv:209]: yeah I think we keep those two

**Review by ZhuTopher — APPROVED** [review]
Thanks, LGTM

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**